linchpin/provision/roles/dependencies/tasks/libvirt_deps.yml

```diff
@@ -0,0 +1,25 @@
+- name: Install dependencies
+  block:
+    - name: Install package dependencies
+      package:
```

Comment:
Please see https://github.com/ansible/ansible/issues/24581

You should not use with_items with package module.  Pass the list directly to name: in package module.

Reply:
@samvarankashyap fyi this appears in a few places